aws-sagemaker-hosted-scorer-cpp/README.md

```diff
@@ -0,0 +1,93 @@
+# DAI Deployment Template for Sagemaker Hosted C++ Scorer
```

Comment:
@Susankha do you want to write Teraform template for stack deployment? it could simplify the workflow.